spec/services/builders/metric_chart/base_spec.rb

```diff
@@ -62,4 +62,28 @@
       end
     end
   end
+
+  describe 'abstract methods' do
+    subject { described_class.send(:new) }
```

Comment:
```suggestion
    subject { described_class.new }
```

Reply:
`new` is a private method for all classes in the `BaseService` hierarchy. That's why we can't do `described_class.new` directly.